docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,93 @@
+# Convert ONNX* RNNT to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction convers conversion of RNN-t model from [MLCommons](https://github.com/mlcommons) repository. 
+If you already have a downloaded MLCommons repository you can go right away to **Step 5**.
+
+**Step 1**. Clone the MLCommons repository for inference:
+```bash
+git clone https://github.com/mlcommons/inference
+```
+
+**Step 2**. create a folder for storing pre-trained model and onnx files:
+```bash
+mkdir rnnt_for_openvino 
+cd rnnt_for_openvino
+```
+
+**Step 3**. Download pre-trained weights for PyTorch implementation from https://zenodo.org/record/3662521#.YG21DugzZaQ:
```

Comment:
I leaved this link since this page provides some explanation about the file. So user can see what is going on there, who are the authors and so on. This can be valuable even If they are on *nix and use wget with the full link. If they are on windows they will just click a button and download the `*.pt` file.